bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookieRecoveryTest.java

```diff
@@ -426,12 +421,7 @@ public void testSyncBookieRecoveryToSpecificBookie() throws Exception {
         writeEntriestoLedgers(numMsgs, 10, lhs);
 
         // Call the sync recover bookie method.
-        BookieSocketAddress bookieSrc = new BookieSocketAddress(InetAddress.getLocalHost().getHostAddress(),
-          initialPort);
-        BookieSocketAddress bookieDest = new BookieSocketAddress(InetAddress.getLocalHost().getHostAddress(),
-          newBookiePort);
-        LOG.info("Now recover the data on the killed bookie (" + bookieSrc + ") and replicate it to the new one ("
-          + bookieDest + ")");
+        LOG.info("Now recover the data on the killed bookie (" + bookieSrc + ") and replicate it to other bookies");
```

Comment:
nit: in the previous log line it was saying that it copied to destination bookie. Here it is saying "replicate it to other bookies", is that correct?

Reply:
yes. because `bookieDest` is actually not used.